File: Class_member/2.rb
class Cs
  def Cs.class_method()
    p "Class method"
  end
  def instance_method()
    p "Instance method"
  end
end

i = Cs.new()
Cs.class_method()
i.instance_method()
# Cs.instance_method() > error
# i.class_method() > error

File: Class_member/2.py
class Cs:
    @staticmethod
    def static_method():
        print("Static_method")
    @classmethod
    def class_method(cls):
        print("Class_method")
    def instance_method(self):
        print("Instance_method")

i = Cs()
Cs.static_method()
Cs.class_method()
i.instance_method()
